cpp/include/options.h

```diff
@@ -11,44 +11,17 @@
 #ifndef CPP_INCLUDE_OPTIONS_H_
 #define CPP_INCLUDE_OPTIONS_H_
 
-#include "jniHelper.h"
-
+#include <wire.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <cstring>
 #include <string>
+#include <map>
 
 using ::jnihelper::JniHelper;
+using ::alluxio::JniObjectBase;
```

Comment:
this statement is redundant since you reference `JniObjectBase` in `namespace alluxio`